ui/src/router/history/HistoryView.js

```diff
@@ -6,11 +6,11 @@ import { ListRouterVersionsView } from "../versions/list/ListRouterVersionsView"
 export const HistoryView = ({ router, ...props }) => {
   return (
     <EuiFlexGroup direction="row">
-      <EuiFlexItem grow={5}>
+      <EuiFlexItem grow={1} className="euiFlexItem--mediumPanel">
```

Comment:
Without the minimum width property applied by  `euiFlexItem--mediumPanel`, Activity Log panel grows to occupy most of the space.